examples/bzlmod/entry_point/BUILD.bazel

```diff
@@ -1,18 +1,38 @@
-load("@pip_39//:requirements.bzl", "entry_point")
+load("@python_versions//3.9:defs.bzl", py_entry_point_binary_3_9 = "py_entry_point_binary")
 load("@rules_python//python:defs.bzl", "py_test")
+load("@rules_python//python:py_entry_point_binary.bzl", "py_entry_point_binary")
 
-alias(
+# We can specifically request the py_entry_point_binary to be running with Python 3.9.
```

Comment:
Please include more documentation of what we are supporting here.  What is an entry_point, and how can a user use this?